lib/utils/error.js

```diff
@@ -2,9 +2,14 @@
 const messages = require('./messages');
 const pipelineMessages = ['Compiling', 'Linting', 'Optimizing'];
 const format = (template, params) => {
-  return template
-    .replace(/#{\s*(\w+)\s*}/g, (_, key) => params[key] || '')
-    .concat(`\n${params.error || ''}`);
+  let withError = false;
+  let message = template.replace(/#{\s*(\w+)\s*}/g, (_, key) => {
+    if (key === 'error') withError = true;
+    return params[key] || '';
+  });
+
+  if (!withError) message += `\n${params.error || ''}`;
+  return message;
```

Comment:
When people see `.concat` they usually think of arrays, not strings. So I would prefer `+=`.